tests/test_dsp.py

```diff
@@ -583,6 +583,51 @@ def test_minimum_phase_multidim():
     npt.assert_allclose(imp_minphase.time, imp_zerophase.time, atol=1e-10)
 
 
+def test_start_ir():
+    n_samples = 2**10
+    snr = 60
+    start_sample = np.array([24])
+
+    ir = pf.signals.impulse(n_samples, delay=start_sample)
+    noise = pf.signals.noise(n_samples, rms=10**(-snr/20))
+
+    start_sample_est = dsp.find_impulse_response_start(ir)
+    npt.assert_allclose(start_sample_est, start_sample, atol=1e-6)
+
+    ir_awgn = ir + noise
+    start_sample_est = dsp.find_impulse_response_start(ir_awgn)
+    npt.assert_allclose(start_sample_est, start_sample, atol=1e-2)
+
+
+def test_start_ir_sinc():
+    sr = 44100
+    n_samples = 128
+    samples = np.arange(n_samples)
+    delay_samples = n_samples // 2 + 1/2
+
+    sinc = np.sinc(samples - delay_samples)
+    win = sgn.get_window('hanning', n_samples, fftbins=False)
+
+    ir = pf.Signal(sinc*win, sr)
```

Comment:
could be done using only `pf.signals.impulse` and `pf.dsp.linear_phase` (see comment for plot above)

Reply:
True, but does not really make a difference I'd say. It's the same code as for the example.

Reply:
On the other hand, for the example, the code is much more compact using the linear phase function. I wanted to be a bit more explicit, but more compact is nicer I guess.